mgr/src/server/server.cr

```diff
@@ -116,18 +116,13 @@ module StorageMgr
     if GlobalConfig.agent
```

Comment:
Remove if condition

Reply:
Sorry for the confusion. We also need else block as existing.

Reply:
Retained else block.